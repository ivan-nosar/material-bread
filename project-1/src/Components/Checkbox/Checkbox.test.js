import React from 'react';
import { Checkbox } from '../..';

import ShallowRenderer from 'react-test-renderer/shallow';
const renderer = new ShallowRenderer();

test('Checkbox Renders', () => {
  const tree = renderer.render(<Checkbox />);
  expect(tree).toMatchSnapshot();
});
