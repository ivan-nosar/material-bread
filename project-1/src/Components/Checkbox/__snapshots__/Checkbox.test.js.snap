// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox Renders 1`] = `
<Context.Consumer>
  [Function]
</Context.Consumer>
`;
